index.html

```diff
@@ -65,26 +65,6 @@
         text-align: right;
       }
 
-      .panel-group .panel-collapse.collapse {
```

Comment:
@brunocodutra Can you confirm this is what you meant? I'm completely illiterate in CSS, so I just followed your instructions blindly.